src/validation-group.js

```diff
@@ -427,7 +427,7 @@ export class ValidationGroup {
    * @param threshold {Object} an optional treshold object that is passed to the conditionExpression
    * @returns {ValidationGroup} returns this ValidationGroup, to enable fluent API
    */
-  if(conditionExpression, threshold) {
+  if(conditionExpression: Function, threshold: any): ValidationGroup {
```

Comment:
instead of `Function` correct type (at least based on documentation) would be `() => boolean`.